src/app/page.tsx

```diff
@@ -28,6 +45,15 @@ export default function Home() {
         </div>
       </div>
 
+      <div className={style.container}>
```

Comment:
This is a pretty small example, so not a big deal in this case, but going forward, make sure to create smaller components such as a Button component or a Counter component to ensure that no single file gets too long